docs/release/release-2.0.rst

```diff
@@ -0,0 +1,40 @@
+===========
+Release 2.0
+===========
+
+New Features
+------------
+* Make presto-admin log and configuration directories configurable. They can be
+  set using the environment variables ``PRESTO_ADMIN_LOG_DIR`` and
+  ``PRESTO_ADMIN_CONFIG_DIR``.
```

Comment:
Some bullet points end in periods, some don't. I would prefer every bullet point to have a period at the end but I don't feel strongly as long as we're consistent.